crates/re_space_view_spatial/src/visualizers/transform3d_arrows.rs

```diff
@@ -50,11 +49,21 @@ impl VisualizerSystem for Transform3DArrowsVisualizer {
         query: &ViewQuery<'_>,
         view_ctx: &ViewContextCollection,
     ) -> Result<Vec<re_renderer::QueueableDrawData>, SpaceViewSystemExecutionError> {
-        let mut line_builder = view_ctx.get::<SharedRenderBuilders>()?.lines();
         let transforms = view_ctx.get::<TransformContext>()?;
 
         let store = ctx.entity_db.store();
         let latest_at_query = re_data_store::LatestAtQuery::new(query.timeline, query.latest_at);
+
+        // Counting all transform ahead of time is a bit wasteful and we don't expect a huge amount of lines from them,
+        // so use the `LineDrawableBuilderAllocator` utility!
+        const LINES_PER_BATCH_BUILDER: u32 = 3 * 32; // 32 transforms per line builder (each transform draws 3 lines)
```

Comment:
Where does 32 come from?

Reply:
it literally says there on the right. We want to allocate space for 32 transforms

Reply:
maybe saying "transforms" is confusing and should say "transform-arrow-triples" or something

Reply:
> it literally says there on the right. We want to allocate space for 32 transforms

Why do we want to allocate 32 and not 33? Is this an arbitrary choice?

Reply:
yup. very arbitrary. 32 is a nice number